dev-libs/qr-code-generator/qr-code-generator-1.8.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="QR Code Generator Library in Multiple Languages"
+
+HOMEPAGE="
+	https://github.com/nayuki/QR-Code-generator
+	https://github.com/EasyCoding/qrcodegen-cmake
+"
+
+SRC_URI="
```

Comment:
Typical layout is to have two blocks with only 1 blank line between them, aka:
```
DESCRIPTION=
HOMEPAGE=
SRC_URI=
S=

LICENSE=
SLOT=
KEYWORDS=
(USE=)
```
skel.ebuild does not really relay this well due to the extra splitting with comments, albeit opening an empty "file.ebuild" with e.g. vim would auto-generate a skeleton that looks like that

Reply:
Yeah, I was thinking of doing it more like that, while pondering about the skel being what it is...